nion/swift/test/DisplayPanel_test.py

```diff
@@ -2564,6 +2564,48 @@ def test_interval_zoom(self):
                 self.assertAlmostEqual(axes.drawn_left_channel, (min(interval) - padding) * 1024)
                 self.assertAlmostEqual(axes.drawn_right_channel, (max(interval) + padding) * 1024)
 
+    def test_stacked_lineplot_interval_zoom(self):
```

Comment:
This test is good, but it is very specific. Can you change the name of the test to reflect that: "test_stacked_lineplot_interval_zoom_with_different_intensity_calibration_scales". Some other tests might be:
- test_stacked_lineplot_interval_zoom_with_different_intensity_calibrations_with_no_selected_intervals
- test_stacked_lineplot_interval_zoom_with_different_intensity_calibrations_with_different_intensity_units
- test_stacked_lineplot_interval_zoom_with_different_intensity_calibrations_with_multiple_intervals